plugins/outputs/instrumental/instrumental_test.go

```diff
@@ -81,7 +82,7 @@ func TestWrite(t *testing.T) {
 }
 
 func TCPServer(t *testing.T, wg *sync.WaitGroup) {
-	tcpServer, err := net.Listen("tcp", "127.0.0.1:8000")
+	tcpServer, err := net.Listen("tcp", "127.0.0.1:8887")
```

Comment:
I would use port `:0` here to let the OS decide which one is available. What do you think?

Reply:
agreed! pushed update